data.go

```diff
@@ -106,6 +106,10 @@ func (r *dataReader) Read(b []byte) (n int, err error) {
 				r.state = stateDot
 				continue
 			}
+			if c == '\r' {
+				r.state = stateCR
+				break
```

Comment:
We should probably keep the CR, no? IOW, remove this `break`, just like in the `stateData` case?

Reply:
Oh no, I'm confused. Nevermind!